docs/dev/project_management.rst

```diff
@@ -8,16 +8,52 @@ management tools as well as `Zenhub <https://www.zenhub.com>`__ which provides
 additional features for sprint planning, task estimation, and progress reports.
 
 -------------------------------------------------------------------------------
-Issues and Project Tracking
+Agile Scrum Kanban Iteration SDLC Process
 -------------------------------------------------------------------------------
-We use `Github issues <https://github.com/catalyst-cooperative/pudl/issues>`__ to
-track bugs, enhancements, support requests, and just about any other work that goes
-into the project. Try to make sure that issues have informative tags so we can find
-them easily.
-
-We use Zenhub Sprints, Epics, and Releases to track our progress. These won't be
-visible unless you have the `ZenHub browser extension
-<https://www.zenhub.com/extension>`__ installed.
+
+We merge code to ``dev`` continuously, but plan work in two-week sprints.
+
+We track work as items (GitHub issues and PRs) in a Catalyst-wide Github Project.
+
+During each sprint, people on each team pull from a team-specific list of
```

Comment:
Should we formalize teams & membership?

Reply:
This feels like something more akin to the handbook. I think we can probably leave that out, at least until we have a better idea of how the Github projects are going to be structured. I kind of feel like the Inframundo tab in the GitHub mega projects should just be called PUDL to avoid confusion.

```
During each sprint, people pull from a list of prioritized & scoped items.
```

Reply:
> This feels like something more akin to the handbook. I think we can probably leave that out, at least until we have a better idea of how the Github projects are going to be structured. 

Makes sense!

> I kind of feel like the Inframundo tab in the GitHub mega projects should just be called PUDL to avoid confusion.

Is "stuff that people on Inframundo are working on" the same thing as "work on PUDL"?

Reply:
As far as I'm concerned, yes